chapter05/sxg/ex_5_1.pl

```diff
@@ -0,0 +1,5 @@
+#!perl
+use warnings;
+use strict;
+
+print reverse <>;

```

Comment:
I think this will read the whole file in first, instead of line-by-line, so it will not work well on large files.